workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/SourceToDestCopyActionNode.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.wipro.ats.bdre.wgen;
+
+import com.wipro.ats.bdre.md.api.GetProperties;
+import com.wipro.ats.bdre.md.beans.ProcessInfo;
+import org.apache.log4j.Logger;
+
+import java.util.Enumeration;
+
+/**
+ * Created by cloudera on 3/31/16.
+ */
+
+
+public class SourceToDestCopyActionNode extends GenericActionNode {
+
+    private static final Logger LOGGER = Logger.getLogger(SourceToDestCopyActionNode.class);
+    private ProcessInfo processInfo = new ProcessInfo();
+    private ActionNode actionNode = null;
+
+    /**
+     * This constructor is used to set node id and process information.
+     *
+     * @param actionNode An instance of ActionNode class which a workflow triggers the execution of a task.
+     */
+    public SourceToDestCopyActionNode(ActionNode actionNode) {
+        setId(actionNode.getId());
+        processInfo = actionNode.getProcessInfo();
+        this.actionNode = actionNode;
+    }
+
+    public ProcessInfo getProcessInfo() {
+        return processInfo;
+    }
+
+
+    public String getName() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)